kt/godot-library/src/main/kotlin/godot/runtime/Bootstrap.kt

```diff
@@ -10,22 +10,12 @@ import godot.util.JVMLogging
 import java.io.File
 import java.net.URL
 import java.net.URLClassLoader
-import java.nio.file.FileSystems
-import java.nio.file.StandardWatchEventKinds
-import java.nio.file.WatchService
 import java.util.*
-import java.util.concurrent.Executors
-import java.util.concurrent.ScheduledExecutorService
-import java.util.concurrent.TimeUnit
 
 
 internal class Bootstrap {
-    private val classRegistries: MutableList<ClassRegistry> = mutableListOf()
     private lateinit var classloader: ClassLoader
```

Comment:
Saving these as explicit class variables is no longer necessary isn't it if you call `doInit` each time anyway?